client/lib/signup/step-actions/index.js

```diff
@@ -39,6 +43,22 @@ import { getSiteId } from 'calypso/state/sites/selectors';
 const Visibility = Site.Visibility;
 const debug = debugFactory( 'calypso:signup:step-actions' );
 
+const isUserAssignedFSEBeta = ( userId ) => {
+	const fse_beta_rollout_percentage_start_flow = 10;
```

Comment:
Can we extract that constant to the global scope and upper case its name? Conventions 😬